src/configs/settings.ts

```diff
@@ -711,35 +712,44 @@ export const symbolLayerConfig = {
 /** Default colors layer fill colors per administrative level */
 export const adminLayerColors = ['black', 'red', 'orange', 'yellow', 'green'];
 
+/** interface describing indicator threshold item */
+export interface IndicatorThresholdItem {
+  color: Color;
+  name: string;
+  orEquals?: boolean;
+  value: number;
+}
+
 /** interface describing threshold configs for IRS report indicators */
 export interface IndicatorThresholds {
-  [key: string]: {
-    color: any;
-    orEquals?: boolean;
-    value: number;
-  };
+  [key: string]: IndicatorThresholdItem;
 }
 
-/** Indicator Thresholds for NA (Namibia) */
-export const indicatorThresholdsNA: IndicatorThresholds = {
+/** IRS Reporting configs */
+export const indicatorThresholdsIRS: IndicatorThresholds = {
```

Comment:
@moshthepitt should this be `IRS_NA` to support different thresholds and colors for different implementations of the IRS intervention type? (Like if Zambia has different thresholds or if Thailand wants to use different colors, etc?)

Reply:
@cKellyDesign yes it should, although right now ZM and NA are the same values so I thought we should do the configurability when we need it.

Reply:
@cKellyDesign thoughts on that?

Reply:
Ah yeah ok sounds good, might as well not over engineer it until it's necessary :)

Reply:
:shipit: